dev-libs/libedit/libedit-20170329.3.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit eutils flag-o-matic toolchain-funcs versionator multilib-minimal
+
+MY_PV=$(get_major_version)-$(get_after_major_version)
+MY_P=${PN}-${MY_PV}
+
+DESCRIPTION="BSD replacement for libreadline"
+HOMEPAGE="http://www.thrysoee.dk/editline/"
+SRC_URI="${HOMEPAGE}${MY_P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~sparc-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+IUSE="static-libs"
+
+DEPEND=">=sys-libs/ncurses-5.9-r3[static-libs?,${MULTILIB_USEDEP}]
+	!<=sys-freebsd/freebsd-lib-6.2_rc1"
+
+RDEPEND=${DEPEND}
+
+S="${WORKDIR}/${MY_P}"
+
+PATCHES=("${FILESDIR}/${P}-ncursesprivate.patch"
+	 "${FILESDIR}/${P}-el_fn_sh_complete.patch")
+
+multilib_src_configure() {
+	ECONF_SOURCE="${S}" econf \
+		$(use_enable static-libs static) \
+		--enable-widec \
+		--enable-fast-install
+}
+
+multilib_src_install() {
+	emake DESTDIR="${D}" install
+	gen_usr_ldscript -a edit
+}
+
+multilib_src_install_all() {
+	einstalldocs
+	prune_libtool_files --all
```

Comment:
find "${D}" -name '*.la' -delete || die

is how we do it these days. It should also save dep on ltprune (eutils previously).